packages/editor-ui/src/components/SqlEditor/SqlEditor.vue

```diff
@@ -20,28 +28,61 @@ import {
 	keymap,
 	lineNumbers,
 } from '@codemirror/view';
-import { MSSQL, MySQL, PostgreSQL, sql, StandardSQL } from '@codemirror/lang-sql';
-import type { SQLDialect } from 'n8n-workflow';
-
+import {
+	keywordCompletion,
+	MSSQL,
+	MySQL,
+	PostgreSQL,
+	schemaCompletion,
+	StandardSQL,
+	MariaSQL,
+	SQLite,
+	Cassandra,
+	PLSQL,
+} from './sql-parser-skipping-whitespace';
+import type { SQLDialect as SQLDialectType } from './sql-parser-skipping-whitespace';
```

Comment:
Would it be better to import the dialect types from `n8n-workflow` to keep them in sync?